src/main/java/com/afornalik/ox/model/board/CheckHorizontally.java

```diff
@@ -0,0 +1,50 @@
+package com.afornalik.ox.model.board;
+
+public class CheckHorizontally implements CheckBoard {
+
+    public final Board board;
+    public final int conditionLength;
+
+    public CheckHorizontally(Board board, int conditionLength) {
+        this.board = board;
+        this.conditionLength = conditionLength;
+    }
+
+
+    //in progress - not work yet
+    @Override
+    public boolean check(int location, FieldStatus fieldStatus) throws OutOfBoardException {
```

Comment:
It is very procedural code and a very long method. When refactoring please try to fix it.

Reply:
I will not refactoring this by now, because this method don't work yet. As a commented above.